source/_integrations/sonarr.markdown

```diff
@@ -17,6 +17,15 @@ ha_integration_type: integration
 
 The Sonarr integration pulls data from a given [Sonarr](https://sonarr.tv/) instance. This integration only supports Sonarr v3 instances.
 
+<div class="note">
+
+  The Integration assumes the default port of 8989 for the given URL.
+  You will have to specify the correct port in the URL if a different port is used.
+  For example for: `` https://sonarr.example.com/ ``
+    you should specify the port at the end :  `` https://sonarr.example.com:443/ ``
+
+</div>
```

Comment:
Does this have to be a big note blog in the introduction?

Reply:
I thing it is important to take the attention of the reader as soon as possible because is see it as an easy and common mistake.

I personally putt it there because that way it will be above the (ADD INTEGRATION ) Button and before the Configuration section, that most users go directly to.

Reply:
>  but this is an easy and quick fix for now.

Well good reason no to do it. Things should be fixed in a proper way, not quick fix in the docs.

Reply:
Done